layouts/blog/single.html

```diff
@@ -1,37 +1,69 @@
 {{ define "main" }}
-    <article class="post" itemscope itemtype="http://schema.org/BlogPosting">
-      <div class="header-content">
-        <h1>{{ .Params.title }}</h1>
-        <p>{{ .Params.description }}</p>
+<article class="post" itemscope itemtype="http://schema.org/BlogPosting">
+  <div class="header-content">
+    <h1>{{ .Params.title }}</h1>
+    <p>{{ .Params.description }}</p>
+  </div>
+  <p class="post-author">
+    {{ (.Params.publishdate).Format "Jan 2, 2006" }} <span> | </span> By {{
+    replace (replace .Params.attribution "(" "- ") ")" "" }}
+  </p>
+  {{ if .Page.Params.target_release }} {{ $page_parts := split
+  .Page.Params.target_release "." }} {{ $page_version := int (index $page_parts
+  0) }} {{ $page_revision := int (index $page_parts 1) }} {{
+  $page_normalized_version := add (add (mul $page_version 1000000) (mul
+  $page_revision 1000)) 0 }} {{ $site_parts := split .Site.Data.args.version "."
+  }} {{ $site_version := int (index $site_parts 0) }} {{ $site_revision := int
+  (index $site_parts 1) }} {{ $site_normalized_version := add (add (mul
+  $site_version 1000000) (mul $site_revision 1000)) 0 }} {{ if gt
+  $site_normalized_version $page_normalized_version }}
```

Comment:
This formatting got changed here and is not readable. Can you remove the unnecessary changes like this?